hack/cluster-up.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+KUBEVIRT_DEPLOY_CDI=${KUBEVIRT_DEPLOY_CDI:-true}
+source ./cluster-up/up.sh
```

Comment:
maybe just run it ? (and then export is needed for the line above)
beside that lgtm

Reply:
`source` runs it, and it makes the export unnecessary. So I don't see the benefit here, can you elaborate?